src/crescendo.ts

```diff
@@ -55,58 +72,62 @@ export class Crescendo extends Note {
     // The height at the open end of the cresc/decresc
     this.height = 15;
 
-    Vex.Merge(this.render_options, {
-      // Extensions to the length of the crescendo on either side
-      extend_left: 0,
-      extend_right: 0,
-      // Vertical shift
-      y_shift: 0,
-    });
+    this.render_options = {
```

Comment:
It seems like we don't use the parent's `render_options`, so we can remove this merge entirely.

Reply:
We cannot do that the compiler complains about missing properties.

Reply:
Ah, because `render_options` is already typed in the parent. In that case, let's rename this to `options`:

```typescript
this.options = {
  extend_left: 0,
  ...
}
```

Reply:
done